lib/zwave/ZwaveDevice.js

```diff
@@ -753,26 +810,40 @@ class ZwaveDevice extends MeshDevice {
 		commandClass.on('report', (command, payload) => {
 			if (command.name !== commandId) return;
 
- 			// Catch central scene echos and (sometimes) failing parser
- 			if (command.name === 'CENTRAL_SCENE_NOTIFICATION') {
-
- 				if (typeof previousSequence !== 'undefined' && payload.hasOwnProperty('Sequence Number') && payload['Sequence Number'] === previousSequence) return;
- 				previousSequence = payload['Sequence Number'];
-
- 				if (payload.hasOwnProperty('Properties1') &&
- 					payload.Properties1.hasOwnProperty('Key Attributes') &&
- 					typeof payload.Properties1['Key Attributes'] === 'number') {
- 					switch(payload.Properties1['Key Attributes']) {
- 						case 0: payload.Properties1['Key Attributes'] = 'Key Pressed 1 time'; break;
- 						case 1: payload.Properties1['Key Attributes'] = 'Key Released'; break;
- 						case 2: payload.Properties1['Key Attributes'] = 'Key Held Down'; break;
- 						case 3: payload.Properties1['Key Attributes'] = 'Key Pressed 2 times'; break;
- 						case 4: payload.Properties1['Key Attributes'] = 'Key Pressed 3 times'; break;
- 						case 5: payload.Properties1['Key Attributes'] = 'Key Pressed 4 times'; break;
- 						case 6: payload.Properties1['Key Attributes'] = 'Key Pressed 5 times'; break;
- 					}
- 				}
- 			}
+			// Catch central scene echos and (sometimes) failing parser
+			if (command.name === 'CENTRAL_SCENE_NOTIFICATION') {
```

Comment:
Why is this re-added?

Reply:
It is not re-added (only code style changed), it was never gone, see my latest comment here: https://github.com/athombv/node-homey-meshdriver/pull/66